### PR TITLE
Edits to style guide

### PR DESCRIPTION
**Pull request by ebgitelman**
Added clarification on future tense, conditional tense, and contractions. Cleaned up the rest of the text to better adhere to our style guide.

### CODE REVIEW

File: advocacy_docs/community/contributing/styleguide.mdx [hunk 1]
```diff
@@ -529,16 +550,15 @@ Which results in:
...
 
 ## Markdown
 
-If table formatting can be kept simple (e.g., basic text formatting and using `<br>` tags for paragraph breaks), create a table using Markdown. This is the preferred table format.
+If table formatting can be kept simple (for example, basic text formatting and using `<br>` tags for paragraph breaks), create a table using Markdown. This is the preferred table format.
 
-To create a table, use pipes (`|`) between columns and at least 3 dashes (`-`) separating the header cells from the body cells. A return denotes the start of the next row. The text within each column does not need to align in order to be rendered correctly, and you can inline Markdown or HTML.
+To create a table, use pipes (`|`) between columns and at least 3 dashes (`-`) separating the header cells from the body cells. A return denotes the start of the next row. The text within each column doesn't need to align to be rendered correctly, and you can inline Markdown or HTML.
 
 We don’t use outer pipes.
```
Comment (ebgitelman) [inline advocacy_docs/community/contributing/styleguide.mdx:563]: Is this true?

File: advocacy_docs/community/contributing/styleguide.mdx [hunk 2]
```diff
@@ -125,12 +125,18 @@ Enter the following information:
...
-Use simple and direct language and keep your sentences short. Avoid combining sentences, which makes the content complicated. Maximum sentence length is 26 words when possible.
+Use simple and direct language and keep your sentences short. Avoid combining sentences, which makes the content complicated. The preferred maximum sentence length is 26 words.
 
 ### Contractions
 
 In keeping with the casual and friendly tone, use contractions. However, use common contractions (isn’t, can’t, don’t). Don't use contractions that are unclear or difficult to pronounce (there’ll).
 
+In new reference material, don't use contractions.
+
+For instructions that concern the safety of a person, equipment, or code, it's okay to spell out the words if you choose to, for example:
```
Comment (gvasquezvargas) [inline advocacy_docs/community/contributing/styleguide.mdx:136]: I am not sure I am using the best examples in my suggestion, but I believe we could be more specific:
```suggestion
For instructions that concern important lifecycle operations of an environment, the functionality of databases, or when warning against actions that could interrupt a procedure, it's okay to spell out the words if you choose to, for example: 
```

File: advocacy_docs/community/contributing/styleguide.mdx [hunk 3]
```diff
@@ -592,9 +618,9 @@ Use images to clarify a topic, but use them only as needed. Images are either:
 
 ## Dates
 
-When specifying dates for human readability, use the DD mmm YYYY format with a short month name in English. Where the date is being used in a column in a table, use a leading 0 on the day of month, e.g. 01 Jan 2024, for easier alignment. 
+When specifying dates for human readability, use the DD mmm YYYY format with a short month name in English. Where the date is being used in a column in a table, use a leading 0 on the day of month for easier alignment,for example, 01 Jan 2024.
```
Comment (gvasquezvargas) [inline advocacy_docs/community/contributing/styleguide.mdx:621]:
```suggestion
When specifying dates for human readability, use the DD mmm YYYY format with a short month name in English. Where the date is being used in a column in a table, use a leading 0 on the day of month for easier alignment, for example, 01 Jan 2024.
```

**Review by gvasquezvargas — COMMENTED** [review]
Left a few minor comments ;)

**Review by gvasquezvargas — APPROVED** [review]
LGTM! Thank youuuu!

File: advocacy_docs/community/contributing/styleguide.mdx (hunk 1 above)
Comment (djw-m) [reply advocacy_docs/community/contributing/styleguide.mdx:563]: Yes.

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**